docusaurus.config.js

```diff
@@ -34,17 +34,17 @@ module.exports = {
                                 'academy$',
                                 ...collectSlugs(`${__dirname}/sources/academy/webscraping`),
                                 ...collectSlugs(`${__dirname}/sources/academy/platform`),
-                            ].join('|'),
+                            ].join('$|'),
                         },
                         {
                             label: 'Tutorials',
                             to: `/academy/tutorials`,
-                            activeBaseRegex: collectSlugs(`${__dirname}/sources/academy/tutorials`).join('|'),
+                            activeBaseRegex: collectSlugs(`${__dirname}/sources/academy/tutorials`).join('$|'),
                         },
                         {
                             label: 'Glossary',
                             to: `/academy/glossary`,
-                            activeBaseRegex: collectSlugs(`${__dirname}/sources/academy/glossary`).join('|'),
+                            activeBaseRegex: collectSlugs(`${__dirname}/sources/academy/glossary`).join('$|'),
```

Comment:
Also, the last regex in the array won't have a `$` at the end, so it might get treated a bit differently... but this is a complete nit IMO :)

Reply:
That is a good point. The main reason I did this is because we had one specific page in the glossary that made both the tutorials and glossary button show as active... I'll fix it so it's consistent

Reply:
Yeah, I was wondering whether there was such a case that warranted this change... good catch :)